introduction.tex

```diff
@@ -1,53 +1,22 @@
 \section{Introduction}\label{section:introduction}
+Proof-of-burn was invented in 2012 by Iain Stewart~\cite{stewart} as a mechanism for destroying cryptocurrency in an irrevocable and provable manner. It has since been used as a consensus mechanism similar to proof-of-stake (Slimcoin~\cite{slimcoin}), as a mechanism for establishing identity (OpenBazaar~\cite{zindros2016trust}), notarization (Carbon dating~\cite{clark2012commitcoin} and OpenTimestamps~\cite{todd2016opentimestamps}) and bootstrapping a new cryptocurrency (Counterparty~\cite{counterparty}).
 
-Over the last 5 years there has been an explosion of innovation through new cryptocurrencies. Unfortunately, it is very hard for a new cryptocurrency to gain traction. Without traction, and the market depth it ensues, a cryptocurrency will have difficulty getting listed in any exchange. But without being listed in any exchange a cryptocurrency cannot gain sufficient traction. The cryptocurrency can only be sold over-the-counter (OTC) by its miners or stakeholders, but the market for OTC trading and especially for an unknown cryptocurrency is traditionally very limited.
-
-This chicken-and-egg situation presents the need for a solution that circumvents exchanges and allows users to acquire the cryptocurrency in a usable manner, in order for it to gain traction. We recommend utilizing proof-of-burn in order to can allow users to obtain capital on a new cryptocurrency, automatically and without trusted third parties, by burning a legacy proof-of-work cryptocurrency.
-
-We wish to allow users to exchange a proof-of-work based cryptocurrency for a new cryptocurrency. We call the former the \emph{source cryptocurrency}, and the blockchain this cryptocurrency is based on the \emph{source blockchain}. We call the latter the \emph{target cryptocurrency} and its corresponding blockchain the \emph{target blockchain}. The target blockchain may support burning from multiple source blockchains.
```

Comment:
I think we need these paragraphs in the consumption section in shortened form.

Reply:
they are moved, not deleted

Reply:
and shortened where appropriate